MetaData/data/MetaConditions/Era2016_RR-17Jul2018_v1.json

```diff
@@ -220,6 +220,11 @@
         "JER_down" : -0.1 
     },
 
+    "MjjRegression" :
+    {
+	"weightFile" : "/src/flashgg/Taggers/data/HHTagger/MjjRegression/TMVA_mjj_reg_model_met_corr_All.xml"
```

Comment:
the XYMETCorr_MCyear flag should go in here and be decoupled from the bregr one

Reply:
Done